project/Dependencies.scala

```diff
@@ -3,6 +3,12 @@ import sbt.{ExclusionRule, _}
 object Dependencies {
   import Dependency._
 
+  val mesosDep = Seq(
+    mesos % "compile" from "file:///Users/tim/src/m8e/mesos/build/src/java/target/mesos-1.9.0.jar",
+    "com.google.protobuf" % "protobuf-java" % "3.5.0" % "compile"
```

Comment:
This is needed because when I included the locally built JAR it was not sourcing the dependencies, so I just had to add it manually. It'll be gone when switching to an actual Mesos build (a tagged, non-moving target published build such as M1 or RC1)

Reply:
Btw, are we blocked by a snapshot release?